protocol/kafka_confluent/v2/protocol.go

```diff
@@ -87,6 +90,9 @@ func New(opts ...Option) (*Protocol, error) {
 	}
 	if p.producer != nil {
 		p.producerDeliveryChan = make(chan kafka.Event)
+		if p.producerEventHandler == nil {
+			p.producerEventHandler = func(ctx context.Context, err kafka.Event) {}
```

Comment:
Please provide a useful default implementation using logger to log errors and handling the various cases simmilar to the example shown in the Confluent issue.

Reply:
Done.